### PR TITLE
feat/aave-avalanche

### PR DESCRIPTION
**Pull request by leodinh**

### CODE REVIEW

**Review by deepcrazy — APPROVED** [review]

**Comment by deepcrazy** [comment]
- Looks fine. But Linting is failing. Please fix it.

**Review by deepcrazy — APPROVED** [review]

**Review by dhruvinparikh — CHANGES_REQUESTED** [review]
- [x] Curate all the token addresses of avalanche like [this](https://github.com/Opty-Fi/defi-legos/tree/release-v0.1/src/ethereum/tokens) (feel free to export all avalanche tokens from `avalanche/tokens/index.ts`)
- [x] Export tokens, pools and  protocols of avalanche like [this](https://github.com/Opty-Fi/defi-legos/blob/release-v0.1/src/ethereum/index.ts) 
- [x] Export * of avalanche like [this](https://github.com/Opty-Fi/defi-legos/blob/release-v0.1/src/index.ts)

**Comment by dhruvinparikh** [comment]
> * [ ]  Curate all the token addresses of avalanche like [this](https://github.com/Opty-Fi/defi-legos/tree/release-v0.1/src/ethereum/tokens) (feel free to export all avalanche tokens from `avalanche/tokens/index.ts`)
> * [ ]  Export tokens, pools and  protocols of avalanche like [this](https://github.com/Opty-Fi/defi-legos/blob/release-v0.1/src/ethereum/index.ts)
> * [ ]  Export * of avalanche like [this](https://github.com/Opty-Fi/defi-legos/blob/release-v0.1/src/index.ts)

.

File: src/avalanche/tokens/index.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import PLAIN_TOKENS from "./plain_etokens";
+
+const tokens = {
+  PLAIN_TOKENS,
```
Comment (dhruvinparikh) [inline src/avalanche/tokens/index.ts]: - [ ] Include aTokens as well. Feel free to follow this approach - https://github.com/Opty-Fi/defi-legos/blob/release-v0.1/src/polygon/tokens/index.ts (all tokens are curated)

File: src/avalanche/tokens/index.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import PLAIN_TOKENS from "./plain_etokens";
```
Comment (dhruvinparikh) [inline src/avalanche/tokens/index.ts]:
```suggestion
import PLAIN_TOKENS from "./plain_tokens";
```

**Review by dhruvinparikh — CHANGES_REQUESTED** [review]

**Review by dhruvinparikh — APPROVED** [review]

**Pull request merged**